force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -317,20 +466,32 @@ public with sharing class ClassesObjectsMethods {
      * @return A string containing the details of the person.
      */
 
+    
     public static List<String> personDetails() {
         List<String> results = new List<String>();
         // Create a new instance of Person class
+        Person person1; 
+
         // Name the person 'John Doe' and set the age to 28
+        person1 = new Person('John Doe', 28); 
 
         // Add the details of the person using the introduceYourself method
+        person1.introduceYourself();
 
         // Get the name of the person using the getter method and add it to the list
+        person1.getName();
 
         // Get the age of the person using the getter method and add it to the list
-        
-        return null; // Replace null with the variable you used to store the result
-    }
+        person1.getAge();
 
+        // Add the details of the person to the list
+        results.add(person1.introduceYourself());
```

Comment:
You are calling all of those methods a second time. Rather than calling them on lines 479 - 485, you can remove those lines and just add them directly to the results List like you're doing here